### PR TITLE
make test skip if insufficient credentials

### PR DESCRIPTION
**Pull request by fiennyangeln**
Bug 1436799

@djmitche Hi Dustin Please review my changes:)

### CODE REVIEW

File: test/helper.js [hunk 1]
```diff
@@ -31,14 +31,24 @@ helper.testaccount = _.keys(cfg.app.azureAccounts)[0];
...
-if (!cfg.pulse.password) {
-  console.log('Skip tests for due to missing pulse credentials; ' +
-              'create user-config.yml');
-  process.exit(1);
-}
+// if (!cfg.pulse.password) {
+//   console.log('Skip tests for due to missing pulse credentials; ' +
+//               'create user-config.yml');
+//   process.exit(1);
+// }
```
Comment (djmitche) [inline test/helper.js]: go ahead and remove this..

**Comment by djmitche** [comment]
I'll merge once you've removed that commented-out code.  Just `git commit --amend` it :)

**Comment by fiennyangeln** [comment]
@djmitche ok! done :D thanks for the guides!!

**Comment by djmitche** [comment]
Rock on -- this is a great first step at getting taskcluster-auth to be friendly to contributors :)

**Pull request merged**